src/RestApi/StoreApi/Controllers/ProductCollectionData.php

```diff
@@ -79,7 +79,7 @@ public function get_item_schema() {
 				),
 				'attribute_counts' => array(
 					'description' => __( 'Returns number of products within attribute terms, indexed by term ID.', 'woo-gutenberg-products-block' ),
-					'type'        => 'object',
+					'type'        => 'array',
```

Comment:
Should we improve the description a bit here?  The current description seems to suggest something different than the actual schema.  Maybe just drop the "indexed by term id" bit and just have "Returns number of products within attribute terms."?